site/kube-addon.md

```diff
@@ -292,3 +292,65 @@ For example,
             - name: IPALLOC_RANGE
               value: 10.0.0.0/16
 ```
+
+## <a name="operations"></a> Operations
+
+Weave Net can be used without Kubernetes, and if you have done this before, you may be familiar with the `weave` CLI.
```

Comment:
This sentence doesn't quite read right to me. 
Maybe just say what's possible without speculating on the users path to get here?